examples/go-tc-counter/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/redhat-et/bpfd/examples/go-tc-counter
+
+go 1.18
```

Comment:
Nit: Andrew's Operator changes are using go 1.19, gobpfd and go-counter are still using 1.18. Is there a way to keep all of these in sync?

Reply:
Hrm There might be some Github action that allows us to do this

Reply:
Go workspaces?